packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeatureTest.java

```diff
@@ -143,6 +143,7 @@ public void setValue_shouldDetermineMeteringRectangleWhenValidBoundariesAndCoord
   }
 
   @Test(expected = AssertionError.class)
+  @SuppressWarnings("try")
```

Comment:
The variable `mockedCameraRegionUtils` was unused, so I chose to suppress the warning for now.

Reply:
What's the warming being supposed here? Any suppression that's not inherently obviously (like an unchecked cast, or deprecation on a clearly named legacy method) should be commented with an explanation.